products/osconfig/api.yaml

```diff
@@ -0,0 +1,825 @@
+                description: |
+                  An exclusive list of patches to be updated. These are the only patches that will be installed using 'zypper patch patch:' command.
+                  This field must not be used with any other patch configuration fields.
+                item_type: Api::Type::String
+          - !ruby/object:Api::Type::NestedObject
+            name: 'windowsUpdate'
+            description: |
+              Windows update settings. Use this setting to override the default Windows patch rules.
+            at_least_one_of:
+              - patch_config.0.reboot_config
+              - patch_config.0.apt
+              - patch_config.0.yum
+              # - patch_config.0.goo
+              - patch_config.0.zypper
+              - patch_config.0.window_update
+              - patch_config.0.pre_step
+              - patch_config.0.post_step
+            properties:
+             - !ruby/object:Api::Type::Enum
+              name: 'classifications'
+              at_least_one_of:
+                - patch_config.0.windowsUpdate.classifications
+                - patch_config.0.windowsUpdate.excludes
+                - patch_config.0.windowsUpdate.exclusive_patches
+              description: |
+                Only apply updates of these windows update classifications. If empty, all updates are applied.
+              values:
+                - :CRITICAL
+                - :SECURITY
+                - :DEFINITION
+                - :DRIVER
+                - :FEATURE_PACK
+                - :SERVICE_PACK
+                - :TOOL
+                - :UPDATE_ROLLUP
+                - :UPDATE
+             - !ruby/object:Api::Type::Array
+              name: 'excludes'
+              at_least_one_of:
+                - patch_config.0.windowsUpdate.classifications
+                - patch_config.0.windowsUpdate.excludes
+                - patch_config.0.windowsUpdate.exclusive_patches
+              description: |
+                List of KBs to exclude from update.
+              item_type: Api::Type::String
+             - !ruby/object:Api::Type::Array
+              name: 'exclusivePatches'
+              at_least_one_of:
+                - patch_config.0.windowsUpdate.classifications
+                - patch_config.0.windowsUpdate.excludes
+                - patch_config.0.windowsUpdate.exclusive_patches
+              description: |
+                An exclusive list of kbs to be updated. These are the only patches that will be updated.
+                This field must not be used with other patch configurations.
+              item_type: Api::Type::String
+          - !ruby/object:Api::Type::NestedObject
+            name: 'preStep'
+            description: |
+              The ExecStep to run before the patch update.
+            at_least_one_of:
+              - patch_config.0.reboot_config
+              - patch_config.0.apt
+              - patch_config.0.yum
+              # - patch_config.0.goo
```

Comment:
Do you want this to stay commented out, or should it be removed (here and elsewhere)?

Reply:
@ndmckinley - thanks for the reminder! I meant to test this out. It's an empty block in the API, so I wasn't sure how to add it in here, but I just made it nested with `enabled`. Not sure if that makes sense, or if that could potentially mess up other downstreams (if they end up using this). I'd appreciate your thoughts/suggestions here!